packages/server/package.json

```diff
@@ -25,20 +25,6 @@
   "publishConfig": {
     "access": "public"
   },
-  "bin": {
-    "basemaps-server": "./bin/basemaps-server.cjs"
```

Comment:
The server still needs this to be bundled, otherwise the server wont be able to start?

Reply:
I will test the cli and see how it goes.

Reply:
The bundle will require the cli.ts which have been removed.

Reply:
@blacha I have revert this back, might leave this later as it is not necessary for the story.